README.md

```diff
@@ -62,3 +62,28 @@ In the spirit of openness we will be tagging issues with the following:
 - **DONE** - We consider this issue as resolved - please review and close it. In case of no further activity this issues will be closed after a week.
 
 - **duplicate** - This issue is already addressed elsewhere, see comment with provided references.
+
+### Pull Requests
+When raising a Pull Request (PR) changing the PACK.xsd schema file, please ensure that the version, date of change, release-notes and 
+documentation get updated consistently throughout your PR. 
+
+Rules: 
+- Breaking/incompatible changes are not accepted and require further discussion
+
+Exceptions: 
+- Extending the `DeviceVendorEnum` does not require a version increment of the schema version.
```

Comment:
This is only half the truth. @KeilChris realized that new vendors might require a new schema version tag to be able to refer to the updated schema.

Reply:
If they use the genpack library, they can point the gen_pack.sh script to read a certain tagged version of the PACK.xsd. If you do not update the number and create a new tagged version, you would need to overwrite the tagged version. Is that possible?

Reply:
Yes, an admin can overwrite existing tags but this is often considered bad practice.

Reply:
I thought so! That's the reason I tend to update the version number.

Reply:
I removed the exceptions section.

Reply:
Thanks! (And I'm sorry)